x/evm/keeper/state_transition.go

```diff
@@ -181,6 +184,16 @@ func (k *Keeper) ApplyTransaction(tx *ethtypes.Transaction) (*types.MsgEthereumT
 
 	res.Hash = txHash.Hex()
 	logs := k.GetTxLogs(txHash)
+
+	if !res.Failed() {
+		err = k.PostTxProcessing(txHash, logs)
+		if err != nil {
+			k.RevertToSnapshot(revision)
+			res.VmError = "failed to process native logs"
```

Comment:
let's make this a constant and move it to `errors.go` so that we can check for this error message on tests

Reply:
done